lib/build-view.js

```diff
@@ -182,22 +185,35 @@ module.exports = (function() {
     this.titleText.text('Aborted!');
   };
 
+  BuildView.prototype._render = function() {
+    var string = this.a2h.toHtml(_.escape(this.buffer.toString('utf8')));
+    _.forEach(this.links, function (link, index) {
+      string = string.replace(link.text, '<a id="' + link.id + '">' + _.escape(link.text) + '</a>');
+    });
```

Comment:
Thank you for the feedback! You are absolutely correct. I have fixed this in the next commit!

Any other issues you can foresee with this? Is there anything I have forgotten?